app/boards/shields/bfo9000/Kconfig.defconfig

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2020 Pete Johanson, Kurtis Lew
```

Comment:
All copyright headers should go to ZMK Contributors, please. (Can be seen for e.g. [here](https://github.com/zmkfirmware/zmk/blob/main/app/boards/shields/jian/jian.dtsi#L2))

Reply:
Updated.